EdgeXConfig/manifest.yaml

```diff
@@ -0,0 +1,17 @@
+updated: "true"
+count: 13
+latestVersion: levski
+versions:
+    - main
```

Comment:
We currently only have five valid versions, and the extra versions do not need to appear in the manifest.
```yaml
versions:
   - levski
   - kamakura
   - jakarta
   - ireland
   - hanoi
```